### PR TITLE
AI Debate app Demo

### PR DESCRIPTION
**Pull request by Sanjay8602**
Description
This pull request introduces a new Streamlit application, AI Debate App, to the existing repository of UnifyAI demos.
Debate-App is a web application that enables a back-and-forth conversation between two Language Models (LLMs) on a topic chosen by the user. Users can select any two models, input a query, and visualize the dialogue between the LLMs in real-time.

Changes
New Files: Added all necessary files for the Debate app, including app.py, requirements.txt, and related assets.

Updated Documentation: Included detailed instructions in the README.md within the AI Debate App directory, outlining how to set up and run the application locally and how to access the hosted version.

Demo Video: Link of demo video is attached in the README file of this PR for better visualization
All the folders or files have been added in Unify

### CODE REVIEW

File: Unify/LLM_Debate/Readme.MD [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# AI Debate App
+
+[demo] (https://github.com/Sanjay8602/demo/assets/121057369/8f98d746-e1f2-497a-b2e2-ecd43e29674b)
+
+<video width="640" height="480" autoplay>
+  <source src="./assets/LLMDebate.mp4" type="video/mp4">
+Your browser does not support the video tag.
+</video>
```
Comment (hello-fri-end) [inline Unify/LLM_Debate/Readme.MD:8]: You're still not following the instructions mentioned in the [template](https://github.com/unifyai/demos/blob/main/template/README_TEMPLATE.md?plain=1#L1-L9). Can you also checkout this [example](https://github.com/unifyai/demos/blob/main/Unify/LLM-Wars/README.md?plain=1#L1-L7) if the template is not clear?

Specifically, you need to place the video folder in the root directory and assets folder.

File: Unify/LLM_Debate/Readme.MD [hunk 2]
```diff
@@ -0,0 +1,47 @@
+# AI Debate App
+
+[demo] (https://github.com/Sanjay8602/demo/assets/121057369/8f98d746-e1f2-497a-b2e2-ecd43e29674b)
```
Comment (hello-fri-end) [inline Unify/LLM_Debate/Readme.MD:3]:
```suggestion
[demo](https://github.com/Sanjay8602/demo/assets/121057369/8f98d746-e1f2-497a-b2e2-ecd43e29674b)
```

**Review by hello-fri-end — CHANGES_REQUESTED** [review]

File: Unify/LLM_Debate/Readme.MD (hunk 1 above)
Comment (Sanjay8602) [reply Unify/LLM_Debate/Readme.MD]: I have change from my account to unify account and link working properly

Comment (hello-fri-end) [reply Unify/LLM_Debate/Readme.MD]: Please look at [this](https://github.com/unifyai/demos/blob/main/Unify/LLM-Wars/README.md?plain=1#L5) again and update accordingly. Thanks!

**Comment by Sanjay8602** [comment]
Can u help regarding static_ format in my  file. Because in folder I dont have any static follder?

**Review by Sanjay8602 — COMMENTED** [review]
changed to static

**Pull request merged**